src/styles/index.scss

```diff
@@ -1,3 +1,117 @@
+@import 'utils/variables';
+@import 'utils/mixin';
+@import 'blocks/stars';
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  .imac {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__description {
+    margin-bottom: 16px;
+
+    .name {
+      font-family: Roboto, sans-serif;
```

Comment:
```suggestion
```
Redundant - it will be inherited

![image](https://github.com/user-attachments/assets/baa8528a-a64b-4184-a68a-ed35f3780a4f)